nemo_skills/prompt/config/lean4/nat-to-lean4-math-llama.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
What's here specific to llama? Should it be named differently? Would be good if you can include 1-2 sentences comments in all yaml files saying what they are supposed to be used for

Reply:
\`\`\`lean4

Reply:
got it, maybe reflect that in the name directly instead of referencing llama